### PR TITLE
Map new int and height providers

### PR DESCRIPTION
**Pull request by YanisBft**

### CODE REVIEW

File: mappings/net/minecraft/util/math/ClampedIntProvider.mapping [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_6334 net/minecraft/util/math/ClampedIntProvider
```
Comment (haykam821) [inline mappings/net/minecraft/util/math/ClampedIntProvider.mapping]: This should be in the `intprovider` package too.

File: mappings/net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider.mapping [hunk 2]
```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_6342 net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider
+	FIELD field_33522 CODEC2 Lcom/mojang/serialization/Codec;
```
Comment (haykam821) [inline mappings/net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider.mapping]:
```suggestion
	FIELD field_33522 CODEC Lcom/mojang/serialization/Codec;
```

File: mappings/net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider.mapping [hunk 3]
```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_6342 net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider
```
Comment (haykam821) [inline mappings/net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider.mapping:1]: What exactly does this mean?

File: mappings/net/minecraft/world/gen/heightprovider/VeryBiasedToBottomHeightProvider.mapping [hunk 4]
```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_6343 net/minecraft/world/gen/heightprovider/VeryBiasedToBottomHeightProvider
+	FIELD field_33527 CODEC2 Lcom/mojang/serialization/Codec;
```
Comment (haykam821) [inline mappings/net/minecraft/world/gen/heightprovider/VeryBiasedToBottomHeightProvider.mapping]:
```suggestion
	FIELD field_33527 CODEC Lcom/mojang/serialization/Codec;
```

**Review by haykam821 — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/util/math/ClampedIntProvider.mapping (hunk 1 above)
Comment (haykam821) [reply mappings/net/minecraft/util/math/ClampedIntProvider.mapping]: Fixed in 6655c973a6182bb8d91d0ab12dd0711fa9741af5

**Comment by Shnupbups** [comment]
damnit i just did this
sniped again

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider.mapping (hunk 3 above)
Comment (YanisBft) [reply mappings/net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider.mapping:1]: I don't know, that's just the name :)

Comment (liach) [reply mappings/net/minecraft/world/gen/heightprovider/TrapezoidHeightProvider.mapping:1]: I guess the cumulative density function has a trapezoid shape?

**Review by liach — APPROVED** [review]

**Review by jaskarth — APPROVED** [review]

**Pull request merged**